custom_components/google_home/api.py

```diff
@@ -152,6 +154,10 @@ async def collect_data_from_endpoints(
         device = await self.update_alarms_and_timers(device)
         device = await self.update_alarm_volume(device)
         device = await self.update_do_not_disturb(device)
+        device = await self.update_bluetooth_list(device)
+
+        await self.request_bluetooth_scan(device)
```

Comment:
Why not execute this inside the `update_bluetooth_list` function instead? We all ways want a fresh device list when requesting what BT devices it sees.